libeos-payg/service.c

```diff
@@ -229,8 +231,9 @@ provider_get_first_enabled_cb (GObject      *source_object,
       return;
     }
 
+  clock = epg_real_clock_new ();
```

Comment:
Won't you also need to feed this into `epg_provider_loader_get_first_enabled_async`?

Reply:
Good catch. I opted to instead change `epg_manager_constructed` to initialize the `clock` member with an `EpgRealClock`, which matches how the other construct-time properties are handled.

Reply:
If you're relying on that defaulting behaviour, why not drop this and pass `NULL`?

Reply:
Pushed a fixup